pages/common/git-squash.md

```diff
@@ -0,0 +1,17 @@
+# git squash
+
+> Squash N last changes up to a ref'ed commit.
+> Part of `git-extras`.
+> More information: <https://github.com/tj/git-extras/blob/master/Commands.md#git-squash>.
+
+- Squash everything from soruce branch or commit reference into HEAD:
```

Comment:
I think we should split this to 2 examples to be more clear. Also I think this description is better
```suggestion
- Squash everything from a specific branch info the current branch:
```